docs/index.rst

```diff
@@ -16,6 +16,10 @@ Contents:
    development
    api
 
+.. raw:: html
+
+   <a class="github-fork-ribbon left-bottom fixed" href="https://github.com/diana-hep/pyhf/fork" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>
```

Comment:
I personally prefer `right-top` over `left-bottom`, but I think this is just a matter of personal preference rather then being right/wrong.
![fork_banner_top-right](https://user-images.githubusercontent.com/5142394/44609558-72875580-a7f8-11e8-924d-e1c3df697ca1.png)

Reply:
I did too, but it'll overlap with the top `view page source` link on smaller windows.

Reply:
I can just do it since it's what most people expect anyway.

Reply:
Hm, I shrunk down a window and I see what you mean. However, the "[view page source](http://diana-hep.org/pyhf/_sources/index.rst.txt)" link is just to the `index.rst` so I don't think that a huge number of people are going to be clicking that (and the link is still accessible to click from under the banner even if you really shrink down the screen size).